pkg/abi/linux/fuse.go

```diff
@@ -686,6 +726,15 @@ func (r *FUSEDirent) MarshalUnsafe(dst []byte) {
 	name.MarshalUnsafe(dst)
 }
 
+// MarshalBytes serializes FUSEDirent to the dst buffer.
+func (r *FUSEDirent) MarshalBytes(dst []byte) {
```

Comment:
Same as above, are MarshalBytes and MarshalUnsafe used?

Reply:
It is used in `NewRequest`. See other comment.